.github/workflows/release.yml

```
@@ -0,0 +1,39 @@
+name: Release Packages
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - packages/sdks/javascript/src/**
+      - packages/sdks/javascript/package.json
+      - packages/ui/react/src/**
+      - packages/ui/react/package.json
```

Perhaps use wildcards so we don't need to update this file every time a new package is added to the repo?

```suggestion
      - packages/**/src/**
      - packages/**/package.json
```